README.md

```diff
@@ -46,52 +46,54 @@ Getting started with the CoVID-19 CLI is very simple. Kindly follow the steps gi
 
 This CLI is very easy to use.
 
-- **`covid19`**  
-  This is the name of the package that is used to invoke the CLI.  
-  The following flag is supported.
+### `covid19`
 
-  |     Flag     | Alias |                Description                 |
-  | :----------: | :---: | :----------------------------------------: |
-  | `---version` | `-v`  | Display the current version of the program |
+This is the name of the package that is used to invoke the CLI.  
+ The following flag is supported.
 
-  Example:
+|    Flag     | Alias |                Description                 |
+| :---------: | :---: | :----------------------------------------: |
+| `--version` | `-v`  | Display the current version of the program |
 
-  ```bash
-  covid19 -v
-  ```
+Example:
 
-  ```bash
-  covid19 --version
-  ```
+```bash
+covid19 -v
+```
 
-- **`covid19 stats`**  
-  This command returns the statistics of CoVID 19 all over the world.
+```bash
+covid19 --version
+```
 
-  Example:
+### `covid19 stats`
 
-  ```bash
-  covid19 stats
-  ```
+This command returns the statistics of CoVID 19 all over the world.
 
-  The following flag is supported.
+Example:
 
-  |            Flag            |       Alias        |                 Description                 |
-  | :------------------------: | :----------------: | :-----------------------------------------: |
-  | `---country <countryName>` | `-c <countryName>` | Display statistics for a particular country |
+```bash
+covid19 stats
+```
 
-  The supported formats for `countryName` are ISO2, ISO3 or a single-word name.
-  <br/>
-  A list of ISO Country Codes can be <a href="https://www.iban.com/country-codes" target="_blank">found here</a>.
+The following flag is supported.
 
-  Example:
+|           Flag            |       Alias        |                 Description                 |
+| :-----------------------: | :----------------: | :-----------------------------------------: |
+| `--country <countryName>` | `-c <countryName>` | Display statistics for a particular country |
 
-  ```bash
-  covid19 stats -c India
-  ```
+The supported formats for `countryName` are ISO2, ISO3 or a single-word name.
+<br/>
+A list of ISO Country Codes can be <a href="https://www.iban.com/country-codes" target="_blank">found here</a>.
 
-  ```bash
-  covid19 stats --country US
-  ```
+Example:
+
+```bash
+covid19 stats -c India
+```
+
+```bash
+covid19 stats --country US
```

Comment:
As I suggested earlier, you probably should use the same country name in both commands. This reduces the chance of confusion with the two alias flags.